src/mainviews/create-job.tsx

```diff
@@ -248,8 +284,11 @@ export function CreateJob(props: ICreateJobProps): JSX.Element {
             formPrefix={formPrefix}
           />
           <props.advancedOptions
+            mode={'CreateJob'}
             model={props.model}
-            modelChanged={props.modelChanged} />
+            modelChanged={props.modelChanged}
```

Comment:
I think these would be better name `(on/handle)ModelChange` and `(on/handle)ErrorsChange`
From a consumer perspective, the name should give an indication of what these props are meant to do.

Reply:
I think that's good. 

I can also change `mode` to `jobsView` for both parity with the declared TypeScript type (`JobsView`) and less ambiguity with `model`.

Reply:
I think that would be good and will cause less confusion.